test/pkg/environment/expectations.go

```diff
@@ -67,3 +105,28 @@ func (env *Environment) EventuallyExpectHealthy(pods ...*v1.Pod) {
 		}).Should(Succeed())
 	}
 }
+
+func (env *Environment) EventuallyExpectHealthyPodCount(selector labels.Selector, numPods int) {
+	Eventually(func(g Gomega) {
+		g.Expect(env.Monitor.RunningPods(selector)).To(Equal(numPods))
+	}).Should(Succeed())
+}
+
+func (env *Environment) EventuallyExpectScaleDown() {
+	Eventually(func(g Gomega) {
+		// expect the current node count to be what it was when the test started
+		g.Expect(env.Monitor.NodeCount()).To(Equal(env.Monitor.NodeCountAtReset()))
+	}).Should(Succeed())
+}
+
+func (env *Environment) ExpectCreatedNodeCount(comparator string, nodeCount int) {
```

Comment:
This is simple enough that it might be better to just inline this in the test.

Reply:
I like the explicit expect statement inside the tests.